packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Safe
+import { GnosisSafe as Safe } from "safe-contracts/GnosisSafe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+// Contracts
+import { EIP712 } from "@solady/utils/EIP712.sol";
```

Comment:
Is this version of this contract audited? If not, are we included this solady contract in audit scope?

Reply:
EIP712 was audited [here](https://github.com/Vectorized/solady/blob/main/audits/ackee-blockchain-solady-report.pdf).

ECDSA was audited [here](https://github.com/Vectorized/solady/blob/main/audits/cantina-solady-report.pdf)

Reply:
The EIP712 audit is from June 2023, our imported commit is from Jan 2024, and EIP712 [was changed](https://github.com/Vectorized/solady/commits/main/src/utils/EIP712.sol) in between those two dates

The ECDSA audit is from Sep 2023, our imported commit is from Jan 2024, and ECDSA [was also changed](https://github.com/Vectorized/solady/commits/main/src/utils/ECDSA.sol) between those dates

I have not looked at the diffs, but it may be easier to just consider including them in audit scope, because this discussion comes up every time we consider Solady, e.g. happened with Initializeable also (solady is also about to get [re-audited](https://x.com/_hrkrshnn/status/1863751810252767295))

Reply:
Gotcha, I am OK to include them in the scope of the audit in that case.

Reply:
We made the decision to move to OZ to avoid this question for now.